src/pages/components/index.html

```diff
@@ -4,7 +4,16 @@ <h1 class="title">Components</h1>
     </div>
 
     <div class="row text input">
-        <!-- Text input here -->
+        <div class="text-input-box">
+            <h3>Inputs</h3>
+
+            <p>Normal</p>
+
+            <div class="forms-input mb-3">
```

Comment:
Where are these classes defined? They are necessary?

Reply:
I made this class according to the example entries on the bootstrap site, but they are not used in sccs modification so I removed it.